### PR TITLE
fix_npu_embeding

### PR DESCRIPTION
**Pull request by Baibaifan**
### PR types

Bug fixes
### PR changes

APIs
### Describe

Fix npu parallel embedding didn't add padding idx.

### CODE REVIEW

**Review by wangxicoding — DISMISSED** [review]
LGTM

File: paddle/fluid/operators/lookup_table_v2_op_npu.cc [hunk 1]
```diff
@@ -40,6 +40,32 @@ class LookupTableV2NPUKernel : public framework::OpKernel<T> {
         platform::errors::InvalidArgument("npu only accept LoDTensor"));
     output_t->mutable_data<T>(ctx.GetPlace());
 
+    // FIXME(baiyangfan) Fix padding_idx grad bugs.
+    if (ctx.Attr<int>("padding_idx") > 0) {
+      auto data_type = static_cast<framework::proto::VarType::Type>(int);
+      LoDTensor tensor_tmp(data_type);
+      auto tmp_shape = make_ddim({1, table_t->dims()[1]});
+      tensor_tmp.mutable_data<T>(tmp_shape, ctx.GetPlace());
+      T value = 1.0;
```
Comment (sandyhouse) [inline paddle/fluid/operators/lookup_table_v2_op_npu.cc]: 0.0？

File: paddle/fluid/operators/lookup_table_v2_op_npu.cc [hunk 2]
```diff
@@ -93,6 +146,29 @@ class LookupTableV2GradNPUKernel : public framework::OpKernel<T> {
...
+    if (ctx.Attr<int>("padding_idx") > 0) {
+      LoDTensor out1;
+      LoDTensor out2;
+      auto out_shape1 =
+          make_ddim({table_grad_t->dims()[0] - 1, table_grad_t->dims()[1]});
+      auto out_shape2 = make_ddim({1, table_grad_t->dims()[1]});
+      out1->mutable_data<T>(out_shape1, ctx.GetPlace());
+      out2->mutable_data<T>(out_shape2, ctx.GetPlace());
+      std::vector<LoDTensor> outputs = {out1, out2};
+      std::vector<int> sections = {table_grad_t->dims()[0], 1};
```
Comment (sandyhouse) [inline paddle/fluid/operators/lookup_table_v2_op_npu.cc]: table_grad_t->dims()[0] -1 ？

File: paddle/fluid/operators/lookup_table_v2_op_npu.cc [hunk 3]
```diff
@@ -82,6 +108,33 @@ class LookupTableV2GradNPUKernel : public framework::OpKernel<T> {
     }
     */
 
+    // FIXME(baiyangfan) Fix padding_idx grad bugs.
+    if (ctx.Attr<int>("padding_idx") > 0) {
+      auto data_type = static_cast<framework::proto::VarType::Type>(int);
+      LoDTensor tensor_tmp(data_type);
+      auto tmp_shape = make_ddim({1, table_grad_t->dims()[1]});
+      tensor_tmp.mutable_data<T>(tmp_shape, ctx.GetPlace());
+      T value = 1.0;
```
Comment (sandyhouse) [inline paddle/fluid/operators/lookup_table_v2_op_npu.cc]: 0.0？

**Pull request closed without merging**